dicom_reader/__init__.py

```diff
@@ -9,10 +10,10 @@ def call_dicom_reader(data_directory, lib_name):
     args_buffer = args_str.encode('utf-8')
     args_ptr = ctypes.c_char_p(args_buffer)
 
-    lib = ctypes.cdll.LoadLibrary(lib_name)
+    lib = ctypes.CDLL(lib_name)  ## changed from ctypes.dll.LoadLibrary --> ctypes.CDLL for MacOS
```

Comment:
What do you think of using something like:

```python
if platform.system() == 'Linux:
    lib = ctypes.cdll.LoadLibrary(lib_name + '.so')
elif platform.system() == 'Darwin':
    lib = ctypes.CDLL(lib_name + '.dylib')
else:
    raise NotImplementedError('call_dicom_reader not implemented for your operating system')
```

The caller of this function should remove the extension if we do it this way.

Reply:
I think this would be cleaner. I will update.

Reply:
@datapythonista  done